### ISSUE TITLE
Full screen displays large menu bar

### ISSUE THREAD

**Issue by Haroenv** [sindresorhus/caprine#5]
![](https://i.imgur.com/VNK0ysG.png)

Also the regular shortcut: <kbd>⌘</kbd><kbd>⌃</kbd><kbd>F</kbd> isn't enabled.

**Comment by sindresorhus** [comment]
Good catch. I've opened a ticket about fullscreen hiding the toolbar: https://github.com/atom/electron/issues/2810.

As for the keyboard shortcut. I guess I'll have to add a menu called `View` and a menu-item `Enter Full Screen` (And `Exit Full Screen` when fullscreen), unless I can convince Electron to support that keyboard shortcut by default: https://github.com/atom/electron/issues/2811.

**Comment by mfc-julius** [comment]
Also, the full screen is not exiting with "Esc" press. I know not all apps do that too (let's say, Chrome doesn't), but I tend to use it myself.

**Comment by sindresorhus** [comment]
@mfc-julius Yup, already covered in https://github.com/atom/electron/issues/2811.

**Comment by sindresorhus** [comment]
In 0.1.2, you can now use <kbd>⌘</kbd><kbd>⌃</kbd><kbd>F</kbd> to toggle fullscreen. The other issue is blocked by Electron.

**Comment by sindresorhus** [comment]
Everything is now resolved in 0.6.0.

**Comment by Haroenv** [comment]
Great!
